### ISSUE TITLE
How do i even build this?

### ISSUE THREAD

**Issue by Merith-TK** [MycroftAI/Mycroft-Android#90]
I am used to installing deps and building in android studio. but in my case for this app, it litterally REFUSES to build, 

I click make project (CTRL+F9) nothing

i click run  (SHIFT+F10)
![image](https://user-images.githubusercontent.com/10422110/72768374-102cb300-3bac-11ea-9795-2dafc14183cf.png)

please provide more instructions on how to build this application, or provide prebuilt APKs

**Comment by krisgesling** [comment]
Hi Merith, 
This is a Community developed and maintained project that is very much pre-release software. It's coordinated largely through the [~Android channel on Mycroft Chat](https://chat.mycroft.ai/community/channels/android), though it has been fairly quiet lately. 

If you have some experience with Android development and are interested in helping out, I'm sure they would appreciate extra assistance.

There is one pre-built apk available through the [releases](https://github.com/MycroftAI/Mycroft-Android/releases) page.

**Comment by Merith-TK** [comment]
Well one problem i have with the mycroft chat, is that i have logged in with my github via the SSO, but it wont let me login to the chat without email/pass, which was not defined in the setup of my account via SSO

**Comment by krisgesling** [comment]
Yeah, unfortunately we haven't yet integrated Chat into our primary SSO, so it still needs a separate username and password. It's on the backlog but we're a small team so I don't think it's going to happen anytime soon.
